packages/backend/server/schema.prisma

```diff
@@ -475,6 +476,36 @@ model AiWorkspaceEmbedding {
   @@map("ai_workspace_embeddings")
 }
 
+enum AiJobStatus {
+  pending
+  running
+  finished
+  claimed
+  failed
+}
+
+model AiJobs {
+  id          String      @id @default(uuid()) @db.VarChar
+  workspaceId String      @map("workspace_id") @db.VarChar
+  blobId      String      @map("blob_id") @db.VarChar
+  createdBy   String?     @map("created_by") @db.VarChar
+  // job type, like "transcription"
+  type        String      @db.VarChar
+  status      AiJobStatus @default(pending)
+  // job result
+  payload     Json        @db.Json
+
+  startedAt  DateTime  @default(now()) @map("started_at") @db.Timestamptz(3)
+  finishedAt DateTime? @map("finished_at") @db.Timestamptz(3)
+
+  // will delete creator record if creator's account is deleted
+  createdByUser User? @relation(name: "createdAiJobs", fields: [createdBy], references: [id], onDelete: SetNull)
```

Comment:
Is it because the AI results will be shared with others, so the related AiJobs data is not deleted when the user is deleted?

Reply:
right, for data that has already been claimed using the AI plan, other users in the same workspace can also query it.